clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -780,7 +782,7 @@ public void run() {
         }
         if (pushJobSetting.sendControlMessagesDirectly) {
           getVeniceWriter(pushJobSetting).broadcastStartOfPush(
-              SORTED,
+              !pushJobSetting.isMemabledBatchWriteForHybridEnabled && SORTED,
```

Comment:
The var name is weird: `isMemabledBatchWriteForHybridEnabled`. Also the negation is not required, right?